phpBB/phpbb/db/extractor/extractor_interface.php

```diff
@@ -0,0 +1,74 @@
+<?php
+/**
+*
+* This file is part of the phpBB Forum Software package.
+*
+* @copyright (c) phpBB Limited <https://www.phpbb.com>
+* @license GNU General Public License, version 2 (GPL-2.0)
+*
+* For full copyright and license information, please see
+* the docs/CREDITS.txt file.
+*
+*/
+
+namespace phpbb\db\extractor;
+
+/**
+* Database extractor interface
+*/
+interface extractor_interface
+{
+	/**
+	* Start the extraction of the database
+	*
+	* This function initialize the database extraction. It is required to call this
```

Comment:
Shouldn't we throw and exception then if any other method is called before this one?